server/libs/common/src/models/asset.ts

```diff
@@ -0,0 +1,33 @@
+import { Exif } from './exif';
+import { SmartInfo } from './smart-info';
+import { Tag } from './tag';
+
+export interface Asset {
```

Comment:
What is the user of `Asset` here comparing to asset related DTOs as well as `AssetEntity`?

Reply:
1. There are benefits to testing and integration if there are no "database specific" references used in the service (`@Entity`).
2. The service is the common point between the controllers and repositories so the interface belongs here.
3. In general the current "dto"s we have are not ideal. They are too coupled to the web/mobile, and not easily interoperable. I think eventually those dtos should be aligned (replaced) with these interfaces.

Reply:
Can we somehow derive the interfaces from eachother to avoid some of the duplication?